rskj-core/src/main/java/org/ethereum/config/SystemProperties.java

```diff
@@ -66,8 +77,6 @@
  * @since 22.05.2014
  */
 public abstract class SystemProperties {
-    private static final Logger logger = LoggerFactory.getLogger("general");
```

Comment:
I would try to undo all this format changes, it is hard to review in that way and it is not following the rest of the classes.

Reply:
solved